loadgen/demos/lon/sut_over_network_demo.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 The MLPerf Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+
+"""
+Python demo showing how to use the MLPerf Inference load generator bindings over the network.
+This part of the demo runs the "demo SUT" which is connected over the network to the LON node.
+A corresponding "demo LON node" with the demo test is implemented in py_demo_server_lon.py.
+
+The SUT is implemented using a Flask server, with dummy implementation of the inference processing.
+Two endpoints are exposed:
+- /predict/ : Receives a query (e.g., a text) runs inference, and returns a prediction.
+- /getname/ : Get the name of the SUT.
+
+The current implementation is a dummy implementation, which does not use 
+a real DNN model, batching, or pre/postprocessing code,
+but rather just returns subset of the input query as a response,
+Yet, it illustrates the basic structure of a SUT server.
+"""
+
+import argparse
+from flask import Flask, request, jsonify
+
+
+app = Flask(__name__)
+
+
+def preprocess(query):
+    """A dummy preprocess."""
```

Comment:
I think it is an overkill to do it in the loadgen side (e.g., for QSL and QDL).
Adding docstring to "all" functions may also be too much, since for some functions (e.g,. HTTP endpoints) it is already very clear where they run.
In addition, it is written in the docstring per file where it runs, 

So, IMO it would be reasonable to add something of this sort only to the docstring of the three dummy functions (`preprocess()`, `dnn_model()`, `postprocess()`) to further clarify

Reply:
Done